client/src/components/FoodCategory/index.js

```diff
@@ -0,0 +1,59 @@
+import Chip from "@mui/material/Chip";
+import Fade from "@mui/material/Fade";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Switch from "@mui/material/Switch";
+
+export default function FoodCategories(foodCategories) {
+  const [checked, setChecked] = React.useState(false);
+
+  const DEFAULT_FOODS = {
+    Halal: false,
+    Chinese: false,
+    Mexican: false,
+    Amongus: false,
+    Imposter: false,
+  };
+
+  const [foodCategory, setFoodCategory] = React.useState(DEFAULT_FOODS);
+  const foods = Object.keys(foodCategory);
+  const selectedFoods = Object.keys(foodCategory).filter(
+    (food) => foodCategory[food]
+  );
+
+  const handleChange = () => {
+    setChecked((prev) => !prev);
+  };
+
+  return (
+    <div>
+      <FormControlLabel
+        control={
+          <Switch checked={checked} onChange={handleChange} color="warning" />
+        }
+        label="Filter Food Categories"
+      />
+      <div>
+        <Fade in={checked}>
+          {foods.map((food, key) => (
+            <div>
+              <Chip
+                key={key}
+                sx={{ m: 0.25 }}
+                color="warning"
```

Comment:
Not a fan of this. Use "primary" instead (it's our theme)

Reply:
Chip not respecting "primary", used hsl(25, 85%, 57.5%) instead atm